aries-site/src/pages/components/selectmultiple.mdx

```diff
@@ -0,0 +1,47 @@
+import { Example } from '../../layouts';
+import { SelectMultipleAnatomyExample, SelectMultipleSimpleExample } from '../../examples';
+
+The SelectMultiple component is typically used as a form element but can also be leveraged for user experiences that include searching or narrowing down data. It contains an optional search field. When within a form, the SelectMultiple will support labels, error and disabled states.
+
+<Example
+  componentName="SelectMultiple"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/selectmultiple/SelectMultipleSimpleExample.js"
+  docs="https://v2.grommet.io/selectmultiple?theme=hpe#props"
+  grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/SelectMultiple/SelectMultiple.js"
+>
+  <SelectMultipleSimpleExample />
+</Example>
+
+## When to use
+
+When multiple selections can be made in a list that contains 5 or more items.
+
+## When not to use
+
+- For lists that contain less than 5 options to choose from, consider using a [CheckBoxGroup](/components/checkboxgroup) instead. 
+- When only a single selection is permitted, use the [Select](/components/select) component instead.
+
+## Anatomy
+
+<Example plain >
+  <SelectMultipleAnatomyExample />
+</Example>
+
+1. **Select**: Displays current input value and control to open/close options drop
+1. **Selection Summary**: Succinct presentation of what a user can do, has done, and/or reset to default state
+1. **Search (optional)**: Helps narrow down or quickly find options corresponding to the entered keyword.
+1. **Help Text (optional)**
+1. **List**: Available options from which the user can choose
+1. **List Item**: Clearly displays available option and its selected state
+
+## Sizing & Scrolling
+
+- Depending on the length of the dropdown list, the height of the SelectMultiple can be increased or decreased as needed with the `dropHeight` prop.
+- For long lists, consider setting a max height to ensure optimum readability and scannability of list items. 
```

Comment:
can we elaborate on `long lists` for guidance what does that mean

Reply:
Same. If someone wants to add suggested content here feel free.

Reply:
I think Aditi was thinking lists with more than 10 items should have a scroll, so long list = more than ten items